knowage-vue/src/modules/documentExecution/dashboard/widget/WidgetEditor/WidgetEditorSettingsTab/common/interactions/crossNavigation/WidgetCrossNavigation.vue

```diff
@@ -1,7 +1,9 @@
 <template>
     <div v-if="crossNavigationModel" class="p-grid p-jc-center p-ai-center p-p-4">
+        <!-- TODO -->
+        {{ crossNavigationModel }}
```

Comment:
Could you please remove the crossNavigationModel logging?

Reply:
Done.